nfs-server-and-provisioner/base/rbac.yaml

```diff
@@ -0,0 +1,65 @@
+kind: ClusterRole
```

Comment:
This should go to `cluster-scope` since it's a resource that is cluster scoped and requires cluster admin permissions to apply.